### PR TITLE
Add too weak custom encryption exception

### PR DESCRIPTION
**Pull request by kizux**
If the encrypt method used in a custom TextEncryptor returns the same unencrypted text then a dedicated exception is thrown

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@kizux Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1119)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1119) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@kizux Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=1119)!

File: spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java [hunk 1]
```diff
@@ -218,6 +220,15 @@ private String stripFormData(String data, MediaType type, boolean cipher) {
 		body.put("description", "No key was installed for encryption service");
 		return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
 	}
+	
+	@ExceptionHandler(EncryptionTooWeakException.class)
+	@ResponseBody
+	public ResponseEntity<Map<String, Object>> encryptionTooWeak() {
+		Map<String, Object> body = new HashMap<String, Object>();
+		body.put("status", "INVALID");
+		body.put("description", "Encryption method provided too weak");
```
Comment (ryanjbaxter) [inline spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java]: I would change the text to say "The encryption algorithm is not strong enough."

**Comment by kizux** [comment]
Dear @ryanjbaxter , text updated according to your review

**Comment by kizux** [comment]
Of couse, done.

**Comment by ryanjbaxter** [comment]
Looks some other commits have sneaked in.

**Comment by kizux** [comment]
Github doesn't refresh the PR, I cleaned my master branch : https://github.com/kizux/spring-cloud-config/commits/master Can you try to refresh the PR from you side by changing the base branch and go back to master ? Not sure it will do the trick but we can give a try.

**Pull request merged**